framework/contracts/native/account-factory/src/commands.rs

```diff
@@ -364,3 +378,28 @@ pub fn execute_update_config(
 
     Ok(AccountFactoryResponse::action("update_config"))
 }
+
+/// Verifies that *sender* is the owner of *nft_id* of contract *nft_addr*
+fn verify_nft_ownership(
+    deps: Deps,
+    sender: Addr,
+    nft_addr: Addr,
+    nft_id: String,
+) -> Result<(), AbstractError> {
+    // get owner of token_id from collection
+    let owner: OwnerOfResponse = deps.querier.query_wasm_smart(
+        nft_addr,
+        &cw721::Cw721QueryMsg::OwnerOf {
+            token_id: nft_id,
+            include_expired: None,
+        },
+    )?;
+    // verify owner
+    ensure_eq!(
+        sender,
+        owner.owner,
+        AbstractError::Ownership(OwnershipError::NotOwner)
```

Comment:
It's the only place where `AbstractError::Ownership` used, should we replace it to something else?

Reply:
Let's try to do your suggestion after moving the gov implementations to std. If this is still the only use of the OwnershipError type then we can remove it like suggested.